src/edu/cornell/mannlib/vitro/webapp/edit/n3editing/configuration/preprocessors/AddProjectUpdatePreprocessor.java

```diff
@@ -28,6 +28,8 @@ public AddProjectUpdatePreprocessor(EditConfigurationVTwo editConfig) {
 
     }
 
+/* FIXME: This doesn't seem to be called
+*/
     @Override
     public void preprocess(MultiValueEditSubmission inputSubmission, VitroRequest vreq) {
```

Comment:
Let's add a System.out statement here and in ``ProcessRdfForm::applyEditSubmissionPreprocessors`` method to see when and if the preprocessor preprocess method is getting invoked.